jit/gen.c

```diff
@@ -118,6 +122,8 @@ typedef void (*gadget_t)(void);
 #define gg(_g, a) do { g(_g); GEN(a); } while (0)
 #define ggg(_g, a, b) do { g(_g); GEN(a); GEN(b); } while (0)
 #define gggg(_g, a, b, c) do { g(_g); GEN(a); GEN(b); GEN(c); } while (0)
+#define ggggg(_g, a, b, c, d) do { g(_g); GEN(a); GEN(b); GEN(c); GEN(d); } while (0)
+#define gggggg(_g, a, b, c, d, e) do { g(_g); GEN(a); GEN(b); GEN(c); GEN(d); GEN(e); } while (0)
```

Comment:
`gggggg` This is getting a little out of control. Just a tad. :D

(no need to change anything, just commenting)

Reply:
We may consider change to variadic macros later. But to my experience some compilers don't like it.